src/formDemoMockResponse.js

```diff
@@ -75,7 +75,7 @@ const metadata = {
       'href': '/api/v2/it_emx_datatypes_TypeTest/meta/integer',
       'fieldType': 'INT',
       'name': 'integer',
-      'label': 'Integer Field',
+      'label': 'Integer Field with rather long description so we can see it wrap',
```

Comment:
To test this i need a long word instead of a long sentence.

Reply:
When i test using a long word at the end of the sentence i think i actually prefer the wrap.

Reply:
Updated the text to add a long word at the end.
A single asterisk on a new line on its own below the label looks a little lost and out of context to me.